pillar.example

```diff
@@ -2,6 +2,7 @@
 firewalld:
   enabled: True
   ipset: True
+  installbackend: False
```

Comment:
@aboe76, how about adding a hash like:

```yaml
firewalld:
  backend:
    manage: true
    pkg: nftables
```

It looks cleaner to me. (probably we can do the same with `ipset`?

Reply:
@javierbertoli yes you are right, can we change it after this merge?

Reply:
Yeah, makes sense.